eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -93,37 +81,27 @@ stages:
                   deploy:
                     steps:
                       - checkout: none
-                      - template: /eng/pipelines/templates/steps/stage-artifacts.yml
-                        parameters:
-                          SourceFolder: ${{parameters.ArtifactName}}-signed
-                          TargetFolder: staging
-                          FileFilter: ${{artifact.name}}.[0-9]*.[0-9]*.[0-9]*
-                      - pwsh: |
-                          Get-ChildItem -Recurse $(Pipeline.Workspace)/staging
-                        workingDirectory: $(Pipeline.Workspace)
-                        displayName: Output Visible Artifacts
                       - task: NuGetCommand@2
                         displayName: 'Publish ${{artifact.name}} package to NuGet.org'
                         inputs:
                           command: push
-                          packagesToPush: '$(Pipeline.Workspace)/staging/**/*.nupkg;!$(Pipeline.Workspace)/staging/**/*.symbols.nupkg'
+                          packagesToPush: '$(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{artifact.name}}/*.nupkg;!$(Pipeline.Workspace)/staging/**/*.symbols.nupkg'
                           nuGetFeedType: external
                           publishFeedCredentials: Nuget.org
                       - task: NuGetCommand@2
                         displayName: 'Publish to DevOps Feed'
                         inputs:
                           command: push
-                          packagesToPush: '$(Pipeline.Workspace)/staging/**/*.nupkg;!$(Pipeline.Workspace)/staging/**/*.symbols.nupkg'
+                          packagesToPush: '$(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{artifact.name}}/*.nupkg;!$(Pipeline.Workspace)/staging/**/*.symbols.nupkg'
                           publishVstsFeed: ${{ parameters.DevOpsFeedID }}
 
           - ${{if ne(artifact.options.skipSymbolsUpload, 'true')}}:
             - deployment: UploadSymbols
               displayName: Upload Symbols to Symbols Server
               condition: and(succeeded(), ne(variables['Skip.SymbolsUpload'], 'true'))
               environment: nuget
-              dependsOn: PublishPackage
+              dependsOn: PublishPackage                          packagesToPush: '$(Pipeline.Workspace)/staging/**/*.nupkg;!$(Pipeline.Workspace)/staging/**/*.symbols.nupkg'
```

Comment:
Looks like a missing newline.

Reply:
Or a rogue copy/paste.

Reply:
Yeah I think that I fat figured the keyboard.